synfig-core/src/synfig/listimporter.cpp

```diff
@@ -71,10 +67,11 @@ Importer(identifier)
 {
 	fps=15;
 
-	ifstream stream(identifier.filename.c_str());
+	FileSystem::ReadStream::Handle stream = identifier.get_read_stream();
 
-	if(!stream)
-	{
+	// TODO(ice0): There is no way to report about error to GUI. We can't
+	//  throw error from constructor (bad practice), and can't return error code.
```

Comment:
Agree. Should we refactor the Importer class to have a `load()` method – that should be called only once?

Reply:
I think yes, this is a good idea. We can back to this problem after 1.4.1 release.